fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -627,13 +615,17 @@ public static FHIRSearchContext parseQueryParameters(Class<?> resourceType, Map<
                         parameter.getValues().addAll(queryParameterValues);
                         parameters.add(parameter);
                     }
-                }
-
+                } // end else
             } catch (Exception e) {
-                throw SearchExceptionUtil.buildNewParseParameterException(name, e);
+                if (lenient) {
+                    // log and continue
```

Comment:
I don't think this should be in an exception block. 
595 should set a boolean indicating a problem with parsing and in the finally block, it should throw or log. 
on 623 the continue is not necessary you are at the end of the branch.

Reply:
there's a lot of places that throw within this try block.  in all cases we want the same behavior...if there was an issue parsing the query parameter then move on to the next one

Reply:
messy... and understandable.  mind adding your comment as a comment in the Exception block.  ?

Reply:
done